pkg/trtl/server.go

```diff
@@ -117,6 +123,14 @@ func (t *Server) Serve() (err error) {
 		go t.replica.AntiEntropy()
 	}
 
+	// If metrics are enabled, start Prometheus metrics server as separate go routine
+	if t.conf.MetricsEnabled {
+		log.Info().Str("listen", t.conf.MetricsAddr).Msg("trtl metrics server started")
```

Comment:
The `serveMetrics` function also has a `log.Info()` - is this log message redundant? If not, should we make one of them a debug message?